contracts/interfaces/access-control/IRBAC.sol

```diff
@@ -7,6 +7,12 @@ import "../../libs/data-structures/StringSet.sol";
  *  @notice The RBAC module
  */
 interface IRBAC {
+    enum PermissionStatus {
```

Comment:
Please move this enum to the contract. It is internal and should not be in the interface

Reply:
done